documentation/ame-guide/modules/ROOT/pages/support/support.adoc

```diff
@@ -16,30 +16,6 @@ To enable the community to give you the best possible advice, please include the
 * Version of the Aspect Model Editor you use.
 * Additional helpful resources (screenshots, TTL files, etc.).
 
```

Comment:
Why this section with the keyboard shortcut to developer tools was deleted? If the app does not behave as expected the user can open developer tools and send us the error if he has any